### PR TITLE
Update peer dependency eslint-plugin-n to v16

### PR DESCRIPTION
**Pull request by CCFenner**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[x] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
update peer dependency

**Which issue (if any) does this pull request address?**
#1142
https://github.com/standard/eslint-config-standard-with-typescript/pull/1135

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
     "@typescript-eslint/eslint-plugin": "^5.43.0",
     "eslint": "^8.0.1",
     "eslint-plugin-import": "^2.25.2",
-    "eslint-plugin-n": "^15.0.0",
+    "eslint-plugin-n": "^16.0.0",
```
Comment (falsepopsky) [inline package.json:63]:
```suggestion
    "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
```

https://github.com/standard/eslint-config-standard/blob/master/package.json

**Review by falsepopsky — CHANGES_REQUESTED** [review]
It's required to first update `eslint-config-standard` to the new version `17.1.0`

**Comment by mightyiam** [comment]
Thank you for this, @CCFenner. This bump should go together with bumping esling-config-standard. In fact, I wonder why no tests failed...

**Comment by mightyiam** [comment]
Thank you for you effort, @CCFenner . This took some more work, so we did this ourselves. Enjoy.

**Pull request closed without merging**